Gemfile.lock

```diff
@@ -0,0 +1,56 @@
+GEM
```

Comment:
I don't think this file needs to be checked in - instead, you should add it to the `.gitignore`

Reply:
OK sure thing